### PR TITLE
2 Chinese languages, dynamic HKD rate

### PR DESCRIPTION
**Pull request by samL88**
as title

### CODE REVIEW

File: calculate.py [hunk 1]
```diff
@@ -64,17 +64,22 @@ def get_10year(lang):
...
-    res = requests.get(url)
-    rates = res.json()
-    last_price = rates[6]
-    sathkd = round((1/last_price)*100000000/7.75)  # assume exch rate is 7.75
+    satDenominator = 100000000
+    btcDataURL = "https://api-pub.bitfinex.com/v2/ticker/tBTCUSD"
+    btcRates = requests.get(btcDataURL).json()
+    btcLastPrice = btcRates[6]
+    
+    hkdDataURL = "https://v6.exchangerate-api.com/v6/22d53ba1468c061792921328/latest/HKD"
```
Comment (bitkarrot) [inline calculate.py:72]: This URL does not work, if its not a free API, we should not use, as page can be reloaded hundreds of times per day based on the way the server is setup

 hkdDataURL = "https://v6.exchangerate-api.com/v6/22d53ba1468c061792921328/latest/HKD"

Comment (bitkarrot) [reply calculate.py:72]: It is better to use a static once-a-day Forex exchange rate instead of trying to get refresh rate every second here.

**Review by bitkarrot — CHANGES_REQUESTED** [review]

File: calculate.py (hunk 1 above)
Comment (samL88) [reply calculate.py:72]: oh... sorry. It works perfectly now

Comment (samL88) [reply calculate.py:72]: The free API is expected for frequent use as HKD exchange band 7.75 - 7.85 is not guaranteed

Comment (bitkarrot) [reply calculate.py:72]: ok

**Review by bitkarrot — APPROVED** [review]

**Pull request merged**